.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,46 @@
+## Describe your changes
 <!-- Please explain the changes you made -->
 
-<!--
-Please, make sure:
-- you have read the contributing guidelines:
-  https://github.com/zama-ai/kms/blob/main/docs/CONTRIBUTING.md
-- you have formatted the code using rustfmt:
-  https://github.com/rust-lang/rustfmt
-- you have checked that all tests pass, by running `cargo test --workspace`
-- you have updated the changelog (if needed):
-  https://github.com/zama-ai/kms/blob/main/CHANGELOG.md
--->
+## Issue ticket number and link
+<!-- Add a reference to the issue fixed if available -->
+
+## Checklist before requesting a review
+- [ ] My PR title follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), e.g. "chore: made key gen consistent with tfhe-rs 1.4".
+- [ ] I have made sunshine tests for all new `pub` methods.
+- [ ] Code comments are in place for public method along with tricky or non-obvious code segments.
+- [ ] I have performed a self-review of my code
+- [ ] I have not used `unwrap`. Or if I have it is only in tests. 
```

Comment:
```suggestion
- [ ] I have not used `unwrap()` or `expect()`, except in clearly documented places or in tests.
```

Reply:
I don't think `unwrap` should be used in production code. `expect` is fine, but `unwrap` does not force documentation unlike `expect`

Reply:
ok fair enough. Updated in 87574926fd21ec383c5d51c2cc8361fe01f2d7e2